src/Notification.php

```diff
@@ -49,6 +51,10 @@ public static function createItem(array $notificationData): Notification
         $notification->eventCode = $notificationData[self::PROPERTY_EVENT_CODE];
         $notification->success = $notificationData[self::PROPERTY_SUCCESS];
 
+        if (isset($notificationData[self::ADDITIONAL_DATA])) {
+            $notification->additionalData = $notificationData[self::ADDITIONAL_DATA];
```

Comment:
Would be good to validate that the additionalData passed here is an associative array.
There's a very big list of [possible additionalData fields](https://docs.adyen.com/development-resources/webhooks/additional-settings) so it might be quite cumbersome to validate the data itself, but at least we can check for array. What do you think?

Reply:
yes good idea